src/dimensionality_reduction/control_methods/random_features/config.vsh.yaml

```diff
@@ -6,9 +6,15 @@ functionality:
   info:
     type: negative_control
     label: Random features
+    paper_name: "Random Features (baseline)"
+    paper_url: "https://openproblems.bio"
+    paper_year: 2022
+    code_url: "https://github.com/openproblems-bio/openproblems"
```

Comment:
The paper name and paper url don't refer to an actual publication. Also, control methods usually don't need these fields.

I'd remove these four lines.